### ISSUE TITLE
The filter option disappeared 

### ISSUE THREAD

**Issue by ShailyTyagi** [zulip/zulip-android#318]
A series of events lead to disappearing of the filter option.It reappeared when i restarted the app after closing it.And it disappears  whenever I follow the same steps I did earlier.
![2017_01_22_13_09_28](https://cloud.githubusercontent.com/assets/18248854/22181007/f065dd9c-e0a7-11e6-8c43-053b0587d9a7.gif)

**Comment by anuradhagupta345** [comment]
@Sam1301 @timabbott @kunall17 What do you think about it?

**Comment by Sam1301** [comment]
@ShailyTyagi Can you please list the steps which lead to disappearing of filter option? Also, be sure to check the options menu for it.

**Comment by ShailyTyagi** [comment]
No,the filter option is not in the option's menu.
i brought the app in landscape mode.I clicked the search button,typed random text and clicked the search button.Then i pressed the back button to close the keypad.I pressed the options menu icon.It had a filter option.Tapped on the screen to close the menu.Then i brought the app back to portrait mode.I pressed the cross which cleared the text.I again pressed the cross.Then pressed the back arrow icon which brought me to messages screen and i could see no filter option.

**Comment by anuradhagupta345** [comment]
@kunall17 What are your views on this ?

**Comment by kunall17** [comment]
@anuradhagupta345 yeah this a bug and i have no clue why is this happening, but this is rare issue to be worked on (seeing the series of events to be performed) we have other much more important issues to be worked on!
